packages/pluggableWidgets/video-player-native/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [4.0.1] - 2024-8-28
+
+### Changed
+
+-   We have updated both `react-native-video` to 6.3.0
```

Comment:
What do you mean by both?

Reply:
I meant make it native project. It was written that way in the previous pull request.

https://github.com/mendix/native-widgets/pull/128/files